### PR TITLE
Moves 'MaxHotBar' setting to Server's PlayerOptions.

### PR DESCRIPTION
**Pull request by Arufonsu**
This Pull Request should solve the issue #896 

Also: 

- Re-ordered Server's PlayerOptions Alphabetically.
- Added an XML documentation comment for the class PlayerOptions.
- Completely Removed 'MaxHotBar' setting from Core's Config.Options.
- Removed some unused imports.

Preview/Example:

imagen

imagen

### CODE REVIEW

File: Intersect (Core)/Config/Options.cs [hunk 1]
```diff
@@ -113,6 +113,8 @@ public class Options
         public static int RequestTimeout => Instance.PlayerOpts.RequestTimeout;
 
         public static int TradeRange => Instance.PlayerOpts.TradeRange;
+        
+        public static int MaxHotbar => Instance.PlayerOpts.MaxHotbar;
```
Comment (Cheshire92) [inline Intersect (Core)/Config/Options.cs:117]: We're trying to do away with top level options, can you please remove this and refer to the option through its lower level version?

**Review by Cheshire92 — CHANGES_REQUESTED** [review]

File: Intersect (Core)/Config/Options.cs (hunk 1 above)
Comment (Arufonsu) [reply Intersect (Core)/Config/Options.cs]: 👍🏽 Commited.

File: Intersect (Core)/Config/Options.cs [hunk 2]
```diff
@@ -29,11 +29,9 @@ public class Options
         [JsonProperty("OpenPortChecker", Order = 0)]
         protected bool _portChecker = true;
 
-        [JsonProperty("MaxClientConnections")]
-        protected int _maxConnections = 100;
+        [JsonProperty("MaxClientConnections")] protected int _maxConnections = 100;
 
-        [JsonProperty("MaximumLoggedinUsers")]
-        protected int _maxUsers = 50;
+        [JsonProperty("MaximumLoggedinUsers")] protected int _maxUsers = 50;
```
Comment (Cheshire92) [inline Intersect (Core)/Config/Options.cs:34]: Is there a reason you've changes these two lines like this?
Nothing around them is touched and I don't see a reason for this change related to the pr.

File: Intersect (Core)/Config/PlayerOptions.cs [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
+        /// </summary>
+        public bool AllowCombatMovement = true;
+
+        /// <summary>
+        /// If true, it will remove the associated exp, otherwise you will lose the exp based on the exp required to level up.
+        /// </summary>
+        public bool ExpLossFromCurrentExp = true;
+
+        /// <summary>
+        /// First thanks for letting me share an idea, of losing XP when dying <3
```
Comment (Cheshire92) [inline Intersect (Core)/Config/PlayerOptions.cs:20]: Descriptive documentation on what this does please.

**Review by Cheshire92 — CHANGES_REQUESTED** [review]

**Comment by Cheshire92** [comment]
Can you also please fix the merge conflicts?

File: Intersect (Core)/Config/Options.cs (hunk 2 above)
Comment (Arufonsu) [reply Intersect (Core)/Config/Options.cs:34]: My bad, i enabled some a feature that "fixes proper spacing and blank lines" at some point here, will undo all these asap.

**Comment by Arufonsu** [comment]
Moved to #928 in order to Make a more clear PR along with the Review Changes.

**Pull request closed without merging**